SPECS/pyproject-rpm-macros/pyproject-rpm-macros.spec

```diff
@@ -87,36 +107,45 @@ cp -p %{sources} .
 %install
 mkdir -p %{buildroot}%{_rpmmacrodir}
 mkdir -p %{buildroot}%{_rpmconfigdir}/azl
-install -m 644 macros.pyproject %{buildroot}%{_rpmmacrodir}/
-install -m 644 pyproject_buildrequires.py %{buildroot}%{_rpmconfigdir}/azl/
-install -m 644 pyproject_convert.py %{buildroot}%{_rpmconfigdir}/azl/
-install -m 644 pyproject_save_files.py  %{buildroot}%{_rpmconfigdir}/azl/
-install -m 644 pyproject_preprocess_record.py %{buildroot}%{_rpmconfigdir}/azl/
-install -m 644 pyproject_construct_toxenv.py %{buildroot}%{_rpmconfigdir}/azl/
-install -m 644 pyproject_requirements_txt.py %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 macros.pyproject %{buildroot}%{_rpmmacrodir}/
+install -pm 644 pyproject_buildrequires.py %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 pyproject_convert.py %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 pyproject_save_files.py  %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 pyproject_preprocess_record.py %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 pyproject_construct_toxenv.py %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 pyproject_requirements_txt.py %{buildroot}%{_rpmconfigdir}/azl/
+install -pm 644 pyproject_wheel.py %{buildroot}%{_rpmconfigdir}/azl/
 
 %check
-pip3 install more_itertools pytest>=3.9 toml 'tox>=3.27.1,<4.0.0'
+%if 0%{?with_check}
```

Comment:
I think this is not needed - `with_check` will always be set, if we're already in the `%check` stage.

Reply:
thanks, will remove